libeos-updater-util/flatpak.c

```diff
@@ -713,11 +714,13 @@ action_node_should_be_filtered_out (JsonNode  *node,
   GList *iter = NULL;
 
   g_return_val_if_fail (is_filtered != NULL, FALSE);
+  g_return_val_if_fail (is_skipped != NULL, FALSE);
 
   /* No filters, so this action cannot be filtered out */
   if (filters_object_node == NULL)
     {
       *is_filtered = FALSE;
+      *is_skipped = FALSE;
```

Comment:
Since `is_skipped` is an error case, it may be better to immediately set the outparam to `FALSE` upon calling the function and only set it to `TRUE` when the error actually happens (as opposed to having to set it in every return location).

Reply:
Actually, maybe instead of having `is_skipped`, it may be better to just propagate the `EOS_UPDATER_ERROR_UNKNOWN_ENTRY_IN_AUTOINSTALL_SPEC` error (or better yet a new error type `EOS_UPDATER_ERROR_UNKNOWN_FILTER_IN_AUTOINSTALL_SPEC`) up to the caller and then let it figure out that it wants to skip the action. That would avoid having a "pseudo-error" case in this function.

Reply:
Good suggestions.